app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
```

Comment:
These queued events might allow linking a metrics ID to a Sentry report, which isn't something we're intentionally doing right now so I'll omit it for the time being.

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f